Source/WebCore/css/CSSBackgroundRepeatValue.cpp

```diff
@@ -44,17 +44,27 @@ Ref<CSSBackgroundRepeatValue> CSSBackgroundRepeatValue::create(CSSValueID repeat
     return adoptRef(*new CSSBackgroundRepeatValue(repeatXValue, repeatYValue));
 }
 
-String CSSBackgroundRepeatValue::customCSSText() const
+template<typename Maker> decltype(auto) CSSBackgroundRepeatValue::serialize(Maker&& maker) const
```

Comment:
How did you decide when to do this dual mode thing and when to just require `StringBuilder`?